src/components/charts/galacticProperties/galactic-properties.module.scss

```diff
@@ -33,72 +34,125 @@
 }
 
 // mini drawer
-aside {
-  .property-item {
-    margin-bottom: -1px;
-    background-color: $neutral10;
-    transition: background-color $timing $duration;
+aside .property-item {
+  margin-bottom: -1px;
+  background-color: $neutral10;
+  transition: background-color $timing $duration;
+
+  .nav-avatar {
+    @include labelPrimary;
+    font-weight: $medium;
+  }
 
-    .nav-avatar {
-      @include labelPrimary;
-      font-weight: $medium;
-      color: $basePrimary;
-    }
+  $i: 0;
+  @each $color in $chartColors {
```

Comment:
This is an interesting loop approach (as oppose to how we handled this in `galaxy-selector.module.scss` with a `@for` loops).  I don't know that I prefer one to the other, but please update one or the other to use the same strategy so we aren't using competing approaches to accomplish the same thing in two separate places.

Reply:
used the @for loop method as it works nicely.